databases/week2/db1w2hw_hyf_lesson2_hwPart1-part3.sql

```diff
@@ -47,12 +47,18 @@ SET
 WHERE
   id = 36;
 -- Change a task status
+  use hyf_lesson2;
 UPDATE
   task
 SET
-  status_id = 3
+  status_id = 3 -- status.name = 'Done';
+  -- should we JOIN here but how? i tried but did not work ?
```

Comment:
you may use Sub Select query inorder to retrieve ID

Reply:
i am happy that i did it Mani, and probably i did it correctly. i had never thought in that way ! many thanks !